docs/decisions/arch-diagrams/refund-flow.mmd

```diff
@@ -0,0 +1,39 @@
+sequenceDiagram
```

Comment:
@attiyaIshaque There are multiple refund flows for mobile:

1: The User unenrolls from the course in LMS, which creates a return object in LMS
2: User opens a dispute in the mobile store, gets a refund, and our CC receives the notification, and it creates a refunded payment transaction object

This sequence diagram does not reflect all of these interactions. Kindly update it

cc: @syedsajjadkazmii @NawfalAhmed